apps/www/src/components/plate-ui/playground-turn-into-dropdown-menu.tsx

```diff
@@ -107,28 +107,24 @@ const defaultItem = items.find((item) => item.value === ELEMENT_PARAGRAPH)!;
 
 export function PlaygroundTurnIntoDropdownMenu(props: DropdownMenuProps) {
   const value: string = useEditorSelector((editor) => {
-    let commonSelection: string | undefined;
-    let uniqueTypeFound = false;
+    let firstNodeType: string = ELEMENT_PARAGRAPH;
+    let allNodesMatchFirstNode = false;
     const codeBlockEntries = getNodeEntries(editor, {
```

Comment:
All looks good now. The only other thing I can see is you might want to change this variable name.

Reply:
done